service/src/main/java/com/ge/predix/acs/monitoring/AbstractCacheHealthIndicator.java

```diff
@@ -35,11 +41,16 @@ public AbstractCacheHealthIndicator(final RedisConnectionFactory redisConnection
 
     @Override
     public Health health() {
-        if (!this.cachingEnabled) {
+        if (!this.decisionCachingEnabled && this.cacheType.equals("decision")) {
             return AcsMonitoringUtilities
                     .health(Status.UNKNOWN, AcsMonitoringUtilities.HealthCode.DISABLED, DESCRIPTION);
-        }
-        if (!this.healthCheckEnabled) {
+        } else if (!this.subjectCachingEnabled && this.cacheType.equals("subject")) {
```

Comment:
No need for multiple if statements since return is the same regardless what cache is disabled.  Need to verify that we specify the cache type when reporting cache as disabled.